### PR TITLE
Logger filtering extra info

### PR DESCRIPTION
**Pull request by ktrzcinx**
Add few extra information about trace filtering system limitations.

### CODE REVIEW

File: developer_guides/debugability/logger/index.rst [hunk 1]
```diff
@@ -194,6 +194,10 @@ A similar example may be prepared for components on a particular pipeline:
   To track verbose message use firmware build with selected "Trace verbose"
   option under "Trace" menu.
 
+Active trave filters are stored in firmware runtime memory, so after firmware
+restart (eg. after power gating in sleep mode) filters settings will be reset.
+Consider disabling power gating during your debug session.
```
Comment (lgirdwood) [inline developer_guides/debugability/logger/index.rst:199]: We should have the instruction for disabling runtime PM here. @plbossart @kv2019i is there a one line command that @ktrzcinx can insert here.

**Review by lgirdwood — CHANGES_REQUESTED** [review]
We need to include the Linux command line for disabling the SOF driver runtime PM. i.e. this guide should be a one stop shop for tracing

File: developer_guides/debugability/logger/index.rst (hunk 1 above)
Comment (xiulipan) [reply developer_guides/debugability/logger/index.rst:199]: @ktrzcinx I now see the debugfs filter is read only. Can we make it to be read/write and restore the filter when we send CTX_RESTORE?

Comment (ktrzcinx) [reply developer_guides/debugability/logger/index.rst:199]: @lgirdwood I've added cmd to disable PM

@xiulipan I would be nice solution to restore filter setup, but there are a one problematic issue. FW doesn't store default filter settings, so to restore them, FW restart is needed, so in situation when logger is run-up twice with different filter setup without FW reboot, then combination of filters will be used - it already works like this, but now user must be aware of this, after introducing this automat, he won't be.

Comment (xiulipan) [reply developer_guides/debugability/logger/index.rst:199]: @ktrzcinx Let's try to create a discussion about this feature update. I think end user will not use this, we can collect feedback from developer.

**Review by lgirdwood — APPROVED** [review]

**Review by deb-intel — APPROVED** [review]
Will make minor grammatical edits in a separate PR.

**Pull request merged**